lib/ansible/modules/cloud/google/gcp_compute_network.py

```diff
@@ -60,14 +49,18 @@
     default: present
   description:
     description:
-    - An optional description of this resource. Provide this property when you create
-      the resource.
+    - An optional description of this resource. Changing this will force the recreation
```

Comment:
On Ansible, can you change this to `output: only`? Ansible doesn't currently support forceNew.